### PR TITLE
Extend AMX deconv int8 to support oscale+eltwise+eltwise post ops pattern

### PR DESCRIPTION
**Pull request by luweizhou2016**
### Details:
* INT8 deconv with int8 ONEDNN can only support one eltwise postops, which cause following pattern degrade to VNNI deconv kernel on SPR.  onednn_verbose,exec,cpu,deconvolution,jit_deconvolution:avx512_core_amx_int8,forward_inference,src_s8::blocked:acdeb:f0 wei_s8::blocked:ABcde16b16a4b:f0 bia_undef::undef::f0 dst_s8::blocked:acdeb:f0,attr-scratchpad:user attr-oscale:2:* attr-post-ops:eltwise_linear:1:0.011765+eltwise_round_half_to_even+eltwise_clip:-3:3 ,alg:deconvolution_direct,mb1_ic320oc320_id7od14kd2sd2dd0pd0_ih7oh14kh2sh2dh0ph0_iw5ow10kw2sw2dw0pw0,0.278076
3d-Unet INT8 perf can increase about %8 on SPR with this change.
*

### Tickets:
 - *https://jira.devtools.intel.com/browse/MFDNN-8869*
 -  *JIRA 85157*

### CODE REVIEW

**Comment by luweizhou2016** [comment]
Combined https://github.com/openvinotoolkit/openvino/pull/13782 to test.

**Comment by dmitry-gorokhov** [comment]
@luweizhou2016 please create corresponding PR in OneDNN fork repo

File: src/plugins/intel_cpu/tests/functional/test_utils/fusing_test_utils.hpp [hunk 1]
```diff
@@ -153,6 +153,11 @@ const auto fusingClamp = fusingSpecificParams{std::make_shared<postNodesMgr>(std
                 return ngraph::builder::makeActivation(cfg.input, cfg.type, ngraph::helpers::Clamp, {}, {3.0f, 6.0f});
             }, "Clamp"}}), {"Clamp"}};
 
+const auto fusingRoundHalfToEven = fusingSpecificParams{std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
```
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/tests/functional/test_utils/fusing_test_utils.hpp:156]: seems to be unused

File: src/plugins/intel_cpu/tests/functional/test_utils/fusing_test_utils.hpp [hunk 2]
```diff
@@ -244,6 +249,22 @@ const auto fusingScaleShift = fusingSpecificParams{ std::make_shared<postNodesMg
                 return std::make_shared<ngraph::opset1::Add>(cfg.input, constNode);
             }, "Add(PerChannel)"}}), {"Add"} };
 
+const auto fusingMultiEltwise = fusingSpecificParams{ std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
```
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/tests/functional/test_utils/fusing_test_utils.hpp:252]: I would say name "fuseClampRoundAddRelu" should be more convenient from developer standpoint

**Comment by luweizhou2016** [comment]
ONEDNNhttps://github.com/openvinotoolkit/oneDNN/pull/163

> @luweizhou2016 please create corresponding PR in OneDNN fork repo

https://github.com/openvinotoolkit/oneDNN/pull/163

**Comment by luweizhou2016** [comment]
@dmitry-gorokhov ，Applied review comments. Thx!

**Review by dmitry-gorokhov — APPROVED** [review]

**Comment by luweizhou2016** [comment]
Combined with https://github.com/openvinotoolkit/openvino/pull/14324

**Pull request merged**